integration/spark/src/main/common/java/io/openlineage/spark/agent/facets/builder/DatabricksEnvironmentFacetBuilder.java

```diff
@@ -0,0 +1,106 @@
+package io.openlineage.spark.agent.facets.builder;
+
+import com.databricks.backend.daemon.dbutils.MountInfo;
+import com.databricks.dbutils_v1.DbfsUtils;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.type.CollectionType;
+import io.openlineage.spark.agent.facets.EnvironmentFacet;
+import io.openlineage.spark.agent.models.DatabricksMountpoint;
+import io.openlineage.spark.api.CustomFacetBuilder;
+import io.openlineage.spark.api.OpenLineageContext;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.function.BiConsumer;
+import org.apache.spark.scheduler.SparkListenerJobStart;
+import scala.collection.JavaConversions;
+
+/**
+ * {@link CustomFacetBuilder} that generates a {@link EnvironmentFacet} when using OpenLineage on
+ * Databricks.
+ */
+public class DatabricksEnvironmentFacetBuilder
+    extends CustomFacetBuilder<SparkListenerJobStart, EnvironmentFacet> {
+  private static HashMap<String, Object> dbProperties;
```

Comment:
I don't think there's any reason to keep this as a static class field any longer.

Reply:
Yes!  You are absolutely right on that one, Mike.  I've removed that modifier in this latest push and dropped a method that was not being used anywhere as well.